Rest/Controller/ContentController.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+/**
+ * This file is part of the EzSystemsRecommendationBundle package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Rest\Controller;
+
+use eZ\Bundle\EzPublishCoreBundle\Imagine\AliasGenerator as ImageVariationService;
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\REST\Server\Controller as BaseController;
+use EzSystems\RecommendationBundle\Rest\Values\ContentData as ContentDataValue;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface as UrlGenerator;
+
+/**
+ * Recommendation REST Content controller.
+ */
+class ContentController extends BaseController
+{
+    /** @var ConfigResolverInterface */
```

Comment:
AFAIK we are omitting full namespace in doc blocks. Maybe EzReco is an exception (after looking at other classes). @andrerom can you confirm the version?